app/controllers/content_management/articles_controller.rb

```diff
@@ -28,6 +28,12 @@ def update
         end
     end
 
+    def destroy
+        @article = Article.find(params[:id])
+        @article.destroy
+        redirect_to root_path
```

Comment:
Now this is fine, but when we merge it with our implementation we should go back to the `content_management_articles_path`

Reply:
Okay!